dev/ScrollViewer/ScrollViewer.cpp

```diff
@@ -825,8 +859,32 @@ void ScrollViewer::OnHideIndicatorsTimerTick(
 {
     SCROLLVIEWER_TRACE_VERBOSE(*this, TRACE_MSG_METH, METH_NAME, this);
 
-    StopHideIndicatorsTimer(false /*isForDestructor*/);
-    HideIndicators(true /*useTransitions*/);
+    ResetHideIndicatorsTimer();
+
+    if (AreScrollControllersAutoHiding())
+    {
+        HideIndicators();
+    }
+}
+
+void ScrollViewer::OnAutoHideScrollBarsChanged(
+    winrt::UISettings const& uiSettings,
+    winrt::UISettingsAutoHideScrollBarsChangedEventArgs const& args)
+{
+    MUX_ASSERT(SharedHelpers::Is19H1OrHigher());
+
+    // OnAutoHideScrollBarsChanged is called on a non-UI thread, process notification on the UI thread using a dispatcher.
+    auto strongThis = get_strong();
```

Comment:
>auto strongThis = get_strong(); [](start = 4, length = 31)

delete this line and do
```
m_dispatcherHelper.RunAsync([auto strongThis = get_strong()]()
```

Reply:
This does not work unfortunately.

Reply:
Whoops, you don't need to specify a type in the lambda capture so just 
```
m_dispatcherHelper.RunAsync([strongThis = get_strong()]()
```

---
In reply to: [286196194](https://github.com/microsoft/microsoft-ui-xaml/pull/731#discussion_r286196194) [](ancestors = 286196194)